PUBLISHING.md

```diff
@@ -0,0 +1,53 @@
+# Publishing new versions of REANA
```

Comment:
Do we need to have this separate file?  What about moving to the usual docs?

Reply:
By usual docs you mean, `README.rst`, DR docs?

Reply:
The latter, or even better `docs.reana.io`, for external collaborators that might be interested (or even involved one day in releasing).

Note also "invenio" typos:

```console
$ git grep invenio
PUBLISHING.md:12:Upgrade the version in the `invenio/Chart.yaml` file:
PUBLISHING.md:32:$ docker run -v /absolute/path/to/reana:/tmp/invenio/ -it quay.io/helmpack/chart-releaser:v0.2.3 /bin/sh
```